packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -83,6 +84,10 @@ describe( 'Post Editor Performance', () => {
 
 	let traceResults;
 
+	beforeAll( async () => {
+		await activateTheme( 'emptytheme' );
```

Comment:
@youknowriad here's where I'm not sure about the choice. For the site editor tests, we've been using the `emptytheme` since https://github.com/WordPress/gutenberg/pull/37446 but for the post editor tests, it was never specified. Post editor specs have been tested against whatever's currently active, which is `twentytwentyone@latest` by default. Should we use `emptytheme` here as well or keep using the 2021?

Reply:
I feels like at least for the sake of this PR we should continue using twentytwentyone but probably by fixing the version.